pkg/cmd/cli/cmd/deploy.go

```diff
@@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
 	deploy_example = `  // Display the latest deployment for the 'database' deployment config
   $ %[1]s deploy database
 
+  // List all deployments for the 'database' deployment config
+  $ %[1]s deploy database -L
```

Comment:
I just don't know why we would want to replicate describe somewhere else.  It's our go to for "get info about this".  This command is about action and maybe a bit of status.

> On May 7, 2015, at 9:23 PM, Fabiano Franz <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/cmd/deploy.go:
> 
> > @@ -26,6 +26,12 @@ NOTE: This command is still under active development and is subject to change.`
> >  	deploy_example = `  // Display the latest deployment for the 'database' deployment config
> >    $ %[1]s deploy database
> >  
> > +  // List all deployments for the 'database' deployment config
> > +  $ %[1]s deploy database -L
> Or maybe describe display the latest 3 in full details, and all others in one line each, like this command is doing atm.
> 
> —
> Reply to this email directly or view it on GitHub.
>